src/benchmarks/pmemobj_gen.cpp

```diff
@@ -542,8 +542,13 @@ pobj_direct_op(struct benchmark *bench, struct operation_info *info)
 	auto *bench_priv = (struct pobj_bench *)pmembench_get_priv(bench);
 	auto *pw = (struct pobj_worker *)info->worker->priv;
 	size_t idx = bench_priv->obj(info->index);
-	if (pmemobj_direct(pw->oids[idx]) == nullptr)
-		return -1;
+	PMEMoid bad = {1, 1};
```

Comment:
Please add a comment describing what's the purpose of calling direct on this `PMEMoid`. Might not be obvious that this is just to invalidate the cache.